flask_monitoringdashboard/core/reporting/questions/median_latency.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+from scipy.stats import median_test
+
+from flask_monitoringdashboard.core.reporting.questions.report_question import (
+    ReportAnswer,
+    ReportQuestion,
+)
+from flask_monitoringdashboard.database import session_scope
+
+from flask_monitoringdashboard.database.request import get_latencies_sample
+
+
+class MedianLatencyReportAnswer(ReportAnswer):
+    def __init__(
+        self,
+        is_significant,
+        comparison_interval_latencies_sample=None,
+        compared_to_interval_latencies_sample=None,
+        percentual_diff=None,
+        comparison_interval_median=None,
+        compared_to_interval_median=None,
+    ):
+        super().__init__('MEDIAN_LATENCY')
+
+        self._is_significant = is_significant
+        self._comparison_interval_latencies_sample = comparison_interval_latencies_sample
+        self._compared_to_interval_latencies_sample = compared_to_interval_latencies_sample
```

Comment:
would the term *baseline* make sense here? if the meaning is the same reading it might be faster because it's a completely different word.

Reply:
Thanks, that's much better! I couldn't come up with a good name...